api/main_endpoints/routes/Auth.js

```diff
@@ -238,4 +240,147 @@ router.post('/validateVerificationEmail', async (req, res) => {
   });
 });
 
+async function updateTokenDb(email) {
+  const token = crypto.randomBytes(32).toString('hex');
+  try {
+    await PasswordReset.updateOne(
+      {
+        email: email
+      },
+      {
+        token: token,
+        createdAt: Date.now()
+      },
+      {
+        upsert: true
+      }
+    );
+    return token;
+  } catch (err) {
+    // console.log(err);
+    return null;
+  }
+}
+
+function generateResetLink(token) {
+  const link = 'http://localhost:3000/reset-password/?token=' + token;
+  return link;
+}
+
+const { sendResetEmail } = require('../../cloud_api/util/mailer');
```

Comment:
you can't import functions from a different api service, as they are seperate entities in prod.

instead u can make a call to the cloud api like [`api/main_endpoints/util/logging-helpers.js`](https://github.com/SCE-Development/Core-v4/blob/b1634aeb0f8808361e75821001aa72531e6d072b/api/main_endpoints/util/logging-helpers.js)

with peripheral apis. lmk if this makes sense